src/components/ProgressPage.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, useEffect, useState } from "react";
+import { Box, styled, Stepper, Typography, Step, StepLabel, StepConnector, StepConnectorProps} from "@mui/material";
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked';
+
+const MemoryTestsLabels = [
+    {
+        "name": "BoCA - Memory (Immediate Recall)",
+        "desc": "BoCA description"
+    },
+    {
+        "name": "Visual Paired Associates",
+        "desc": "VPA desc"
+    },
+    {
+        "name": "Choice Reaction Time",
+        "desc": "CRT desc"
+    },
+    {
+        "name": "Digit Symbol Matching",
+        "desc": "DSM desc"
+    },
+    {
+        "name": "NeurCare - Spatial Memory",
+        "desc": "Spatial Memory desc"
+    },
+    {
+        "name": "BoCA - Memory Recall (Delayed Recall)",
+        "desc": "BoCA delayed recall desc"
+    }
+];
+
+
+interface ProgressPageProps  {  
+    id:number
+}
+
+interface CustomConnectorProps extends StepConnectorProps {
+    numsteps: number;
+  }
+
+const CustomConnector = styled(StepConnector)<CustomConnectorProps>(({ numsteps }) => ({
+    '& .MuiStepConnector-line': {
+      minHeight: `calc(100vh / ${numsteps} - 20px)`,  // 72px accounts approximately for the height of the step button and label
```

Comment:
Not sure about using `vh` here. The stepper looks too big on mobile device. Need to zoom out in the page, otherwise, won't be able to scroll down to press "continue".

Reply:
adjusted stepper sizes to allow users to press continue button